roles/confluent.variables/vars/main.yml

```
@@ -283,10 +296,19 @@ schema_registry_properties:
       confluent.schema.registry.auth.mechanism: JETTY_AUTH
       public.key.path: "{{rbac_enabled_public_pem_path}}"
       authentication.roles: "**"
-      confluent.metadata.bootstrap.server.urls: "{{mds_urls}}"
+      confluent.metadata.bootstrap.server.urls: "{{mds_bootstrap_server_urls}}"
       confluent.metadata.http.auth.credentials.provider: BASIC
       confluent.metadata.basic.auth.user.info: "{{ schema_registry_ldap_user | default('MISSING') }}:{{ schema_registry_ldap_password | default('MISSING')}}"
 
+  rbac_external_client:
+    enabled: "{{rbac_enabled and external_mds_enabled and mds_broker_listener.ssl_enabled}}"
```

```suggestion
    enabled: "{{rbac_enabled and external_mds_enabled and mds_ssl_enabled}}"
```
and if mds only has one way TLS then shouldnt the SR only need a truststore?